NEW: Allow search field customisation (#9341)
silverstripe/silverstripe-framework: docs/en/02_Developer_Guides/00_Model/11_Scaffolding.md

```

### Primary search field

> In tabular views such as `GridField`, there is a monolithic search bar. By default, this will be the *first

Reviewer: ```suggestion
Tabular views such as `GridField` or `ModalAdmin` include a search bar. By default, the search bar will search against the *first
```
